rcjaRegistration/regions/utils.py

```diff
@@ -0,0 +1,28 @@
+from django.db.models import Q
+
+from regions.models import State, Region
+
+def getRegionsLookup():
+    """
+    Creates a list of objects with state id and a list of regions that are available for that state.
+    """
+    def createLookupObj(stateID, regions):
+        class RegionLookupObj:
```

Comment:
It's worth noting that by defining a function scope local class, we re-instantiate the class every time this function is run, which has a perf penalty (probably irrelevant, but worth noting)

Reply:
Will change that